.gitmodules

```diff
@@ -16,9 +16,6 @@
 [submodule "sm/iproute2"]
 	path = sm/iproute2
 	url = git://git.kernel.org/pub/scm/network/iproute2/iproute2.git
-[submodule "iproute2"]
-	path = sm/iproute2
-	url = git://git.kernel.org/pub/scm/network/iproute2/iproute2.git
-[submodule "ethtool"]
+[submodule "sm/ethtool"]
```

Comment:
Is renaming it from *ethtool* to *sm/ethtool* intentional and going to cause problems for existing checkouts?

Reply:
Hi Paul,

I accidentally added the submodule wrong, they should all be in sm/.  There should be no issue with the change afaik, but I will pull and check again.

Reply:
Oh, it’s a fixup commit. Sorry, I missed that. Next time, you can just squash both commits.